java/stop.sh

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
+
+bin=`dirname ${0}`
+bin=`cd ${bin}; pwd`
+basedir=${bin}/..
+RUN_DIR=${basedir}/run
```

Comment:
`run_pids`

Reply:
...I need a directory to store all processes' pid, the file name is the process name... :-)

Reply:
em, Is `run_pids` as the folder name more suitable than `run`?

Reply:
`run` is fine as well.